linkme-chatapp/src/index.js

```diff
@@ -39,7 +39,7 @@ window.addEventListener('message', function(evt) {
     $.ajax({
```

Comment:
I wonder if we can use request here instead of $.ajax. I am not entirely sure because this is in the frontend.